packages/warehouses/src/warehouseClients/SnowflakeWarehouseClient.ts

```diff
@@ -276,7 +288,7 @@ export class SnowflakeWarehouseClient extends WarehouseBaseClient<CreateSnowflak
                             }),
                             {},
                         );
-                        resolve({ fields, rows: parseRows(data) });
+                        resolve({ fields, rows: data });
```

Comment:
we don't need to parse these results as it is only used to fetch the catalog and such.